app/views/overview/_service-instance-row.html

```diff
@@ -14,13 +14,18 @@
           </div>
         </h3>
         <div class="status-icons">
+          <!--
+            TODO: this should show an info status icon if the apiObject is not yet ready.
+          -->
           <notification-icon ng-if="!row.expanded" alerts="row.notifications"></notification-icon>
+          <span ng-if="!row.isReady">
+            <status-icon status="row.status"></status-icon> {{row.statusReason | sentenceCase}}
```

Comment:
it is unlikely status-icon is going to work with the condition reasons we are getting back, and at any time they might add more and break us / cause us to show the ?  I would prefer we check for known failure conditions.